molgenis-core-ui/src/main/resources/css/themes/bootstrap-molgenis.css

```diff
@@ -1,20 +0,0 @@
-.panel-title {
-    color: white;
-}
-
-.panel-heading {
-    background-color: #5B82A4;  
-}
-
-#plugin-container {
-	background: #F8FAFB;
-	padding: 5px;
-}
-
-label.horizontal-inline {
-	margin-right: 5px;
-}
-
-label.horizontal-inline.inbetween {
-	margin-left: 5px;
-}
```

Comment:
I think I got the deduplication running in a different way, plus added unit tests for the style service. What do you think of https://github.com/Mark-de-Haan/molgenis/pull/10 ?